packages/tldraw/src/lib/tools/SelectTool/childStates/Idle.ts

```diff
@@ -261,7 +261,7 @@ export class Idle extends StateNode {
 						info.handle === 'top' ||
 						info.handle === 'bottom'
 					) {
-						const change = util.onDoubleClickEdge?.(onlySelectedShape)
+						const change = util.onDoubleClickEdge?.(onlySelectedShape, { edge: info.handle })
```

Comment:
i'm confused on our usage of `onDoubleClickEdge`.
for example, in `ImageShapeUtil→onDoubleClickEdge` we create a partial update but just do the update there, feels ilke we should be returning the update, yeah?

oh, i see. it's used in Crop/children/Idle.ts so there's some differences on how we're using it.

anyway, i guess it's fine, might be worth cleaning up now or later, up to you.